src/graph.rs

```diff
@@ -628,12 +628,15 @@ pub struct Import {
 }
 
 #[derive(Debug, Default, Clone, Serialize)]
+#[cfg_attr(test, derive(Eq, PartialEq))]
```

Comment:
Maybe we should just add this unconditionally so we can also potentially use it in the cli?